Observer/OrderSaveAfter.php

```diff
@@ -57,22 +59,26 @@ function execute(Observer $ob) {
 							? '{"error": "Magento webhook failed to encode order, please look at order ' . $order_id . ' on website to see the details."}'
 							: $encoded_str;
 						$res = $this->post($payload_str, $o->getStore());
+					} catch (OrderLineError $e) {
+						$comment[] = 'Stock2Shop Webhook exception: ' . $e->getMessage();
 					} catch (\Exception $e) {
```

Comment:
Let's just change this to Throwable and revert the rest (and retest)

Reply:
[fc29aab](https://github.com/stock2shop/magento2_module_webhook/pull/12/commits/fc29aab9979688acfdba83c1b29a3f46e690998e)